fdbserver/LogRouter.actor.cpp

```diff
@@ -75,20 +75,22 @@ struct LogRouterData {
 
 	UID dbgid;
 	Reference<AsyncVar<Reference<ILogSystem>>> logSystem;
-	NotifiedVersion version;
-	NotifiedVersion minPopped;
+	NotifiedVersion version; // The largest version at which the log router has peeked mutations from satellite tLog.
```

Comment:
I am not sure if we want to mention `satellite tLog` in log router, since, although it prefers to peek from satellite, it can totally do that from primary logs if `LOG_ROUTER_PEEK_FROM_SATELLITES_PREFERRED` is set to 0.